<launch>
 <!-- The name of the parameter under which the URDF is loaded -->
  <arg name="robot_description" default="robot_description"/>
  <arg name="trajopt_description" default="trajopt_description"/>
  <arg name="plotting" default="true"/>

  <!-- Load universal robot description format (URDF) -->
<!--  <param name="$(arg trajopt_description)" textfile="$(find trajopt_examples)/config/basic_cartesian_plan.json"/>-->

  <!-- Load universal robot description format (URDF) -->
  <param name="$(arg robot_description)" command="$(find xacro)/xacro --inorder '$(find trajopt_examples)/urdf/car_seat_demo.xacro'" />

  <!-- The semantic description that corresponds to the URDF -->
  <param name="$(arg robot_description)_semantic" textfile="$(find trajopt_examples)/config/car_seat_demo.srdf" />

  <!-- If needed, broadcast static tf for robot root -->
<!--  <node pkg="tf" type="static_transform_publisher" name="virtual_joint_broadcaster_0" args="0 0 0 0 0 0 world base_link 100" />-->

  <node pkg="trajopt_examples" type="trajopt_examples_car_seat_demo" name="trajopt_examples_car_seat_demo">
    <param name="plotting" type="bool" value="$(arg plotting)"/>
  </node>
</launch>

